client/rest/cmd.go

```diff
@@ -0,0 +1,45 @@
+package rest
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/spf13/cobra"
+
+	coinrest "github.com/tendermint/basecoin/modules/coin/rest"
+)
+
+var ServeCmd = &cobra.Command{
+	Use:   "serve",
+	Short: "Serve the light REST client for tendermint",
+	Long:  "Access basecoin via REST",
+	RunE:  serve,
+}
+
+var port int
+
+func init() {
+	ServeCmd.PersistentFlags().IntVar(&port, "port", 8998, "the port to run the server on")
```

Comment:
Yeah Int() doesn't work alright, I think I mentioned this offline, it requires me then to invoke
ServeCmd.Parse(args) of which args aren't available until after we've entered the command .RunE and even then it isn't playing well with the flags cobra passes in []string{}. I spent too much on this that I figured I'd rather make functionality changes than be stuck trying to debug ServeCmd.